portfolio/src/main/webapp/index.html

```diff
@@ -17,20 +17,19 @@ <h1>Portfolio</h1>
         <strong><div id="summary"> </div>Hello, My name is<div id="picture"></div> </strong>
         <h1>Andre Battle</h1>
         <p>Click continue to see content</p>
-        <button class="button" onclick="showNext()">Continue</button>
+        <button id="content-button" class="button" onclick="showNext()">Show Content</button>
+        <br></br>
         <button class="button" onclick="showComments()">Show Comments</button>
+        <hr id="spacer">
         <form name="form1" action="/data" method="POST" onsubmit="required(event)">
-            <strong>Please enter your name</strong> 
+            <strong>Please enter your name: </strong> 
+            <input  type="text" name="user-name" placeholder="Your name..">
             <br/></br>
-            <input  type="text" name="user-name">
+            <strong>Leave a thoughtful comment: </strong> 
+            <input type="text" name="user-comment" placeholder="Leave a comment..">
             <br/></br>
-            <strong>Leave a thoughtful comment</strong> 
-            <br/></br>
-            <input type="text" name="user-comment">
-            <br/></br>
-            <input type="submit"/>
+            <input type="submit" value="Add Comment"/>
```

Comment:
Tiny nitpick that I only mention because you're working on cosmetics: hovering over this button pushes the other elements around slightly, I think because it's adding/removing a border which makes the button taller/shorter.